CONTRIBUTING.md

```diff
@@ -244,7 +244,21 @@ The following example is for releasing a patch for `v0.69.0` -> `v0.69.1`.
 - Create PRs with base set to that release branch
   - When the PR is merged, a changeset PR is created
   - When the changeset PR is merged into the release branch, the next patch version is released and the commit is tagged (e.g. `v0.69.1`)
-- After release, delete the release branch from GitHub
+- After release, the release branch will be automatically deleted
+
+# Patching latest release
+
+Imagine the scenario:
+
+1. We release `v0.80.0`
+1. One day later, we have a new changesets PR that will bump things to `v0.81.0`
+1. Before releasing `v0.81.0`, we find an issue and need to make a `v0.80.1` patch
+
+We'd follow the same approach as explained in the [Patching old releases](#patching-old-releases) section above, bearing in mind the following after the release:
+
+- Update the package versions on `master` to match the released versions,
+- If the released commits' functionality is not on `master`, you have to manually add it via a new PR to `master`
+- If the released commits' functionality is already on `master`, you don't have to do anything
```

Comment:
In this scenario, updating `master` should be mandatory, no?

Maybe the guide could explain everything til the end.

Reply:
I explained that the versions have to be updated in the first bullet point (meaning it's mandatory), but if the released functionality already exists on `master` then we don't have to do anything on that front. What else would you add/rephrase here?

Reply:
I think it's not obvious what "versions should be updated" means.

Is it saying that one must re-open their PR against `master`?

Reply:
This part got changed because I automated the PR creation as mentioned in https://github.com/FuelLabs/fuels-ts/pull/2497#discussion_r1639851406.